### PR TITLE
Add contributing docs

### PR DESCRIPTION
**Pull request by adborden**
Please review these and add any comments/questions. Once you're satisfied, please use the Github review tool to Approve the PR. We'll wait on everyone's :white_check_mark: before merging.

### CODE REVIEW

File: team-practices.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# FOIA Discovery Team Practices
```
Comment (pkarman) [inline team-practices.md]: zap the `Discovery` from the title?

File: team-practices.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+This document describes the conventions the FOIA Discovery team has agreed to work under.
```
Comment (pkarman) [inline team-practices.md]: zap `Discovery`

**Comment by adborden** [comment]
ping @AvivaOskow

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+As a work of the United States Government, this project is in the public domain within the United States.
+
+## Restrictions on FEC data
```
Comment (bsweger) [inline LICENSE.md]: Should this be FOIA instead of FEC?

File: LICENSE.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+A few restrictions limit the way you can use FEC data. For example, you can’t use contributor lists for commercial purposes or to solicit donations. [Learn more on FEC.gov.](http://www.fec.gov/pages/brochures/saleuse.shtml)
```
Comment (bsweger) [inline LICENSE.md]: Same note here re: FOIA/FEC.

File: team-practices.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
+# FOIA Team Practices
+
+This document describes the conventions the FOIA team has agreed to work under.
+For information about licensing, code of conduct and contributing to this project,
+see the [CONTRIBUTING](CONTRIBUTING.md) file.
+
+## Sprints
+
+We do one-week sprints, with a retro and planning meeting every Monday, and a sprint review
```
Comment (bsweger) [inline team-practices.md]: Is this true, or do we do two week sprints?

File: LICENSE.md (hunk 3 above)
Comment (adborden) [inline LICENSE.md]: I think this was a copy/paste issue, the paragraph should be dropped. I'll fix it...

File: team-practices.md (hunk 5 above)
Comment (adborden) [inline team-practices.md]: I thought we were doing 1 week sprints, that's what we did in discovery. I think Acquia was thinking two-week, but we need to clarify that. I'll bring it up in slack.

**Comment by adborden** [comment]
@bsweger thanks for the comments, I'll get these fixed ASAP.

File: team-practices.md (hunk 5 above)
Comment (bsweger) [inline team-practices.md]: @adborden Thanks for tracking this down!

**Pull request merged**